cli/user/src/test/java/io/pravega/cli/user/AbstractTlsUserCommandTest.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.user;
+
+import io.pravega.cli.user.config.InteractiveConfig;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.local.LocalPravegaEmulator;
+import io.pravega.test.common.SecurityConfigDefaults;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.Timeout;
+
+import java.net.URI;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class AbstractTlsUserCommandTest {
+
+    // Security related flags and instantiate local pravega server.
+    protected static final AtomicReference<InteractiveConfig> CONFIG = new AtomicReference<>();
+
+    private static final Integer CONTROLLER_PORT = 9090;
+    private static final Integer SEGMENT_STORE_PORT = 6000;
+    private static final Integer REST_SERVER_PORT = 9091;
+
+    @Rule
+    public final Timeout globalTimeout = new Timeout(120, TimeUnit.SECONDS);
+
+    protected boolean authEnabled = false;
```

Comment:
final

Reply:
`authEnabled` and `tlsEnabled` cannot be `final` as they will be changed according to the tests in which they are used. Explained above